src/platforms/dotnet/guides/google-cloud-functions/index.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: Google Cloud Functions
+description: "Learn about Sentry's .NET integration with Google Cloud Functions."
+redirect_from:
+  - /platforms/dotnet/google.cloud.functions/
+---
+
+Sentry provides an integration with `Google Cloud Functions` through the [Sentry.Google.Cloud.Functions NuGet package](https://www.nuget.org/packages/sentry.google.cloud.functions).
+
+## Overview of the Features
+
+- Easy Google Cloud Functions integration, single line: `UseSentry`
+- Captures unhandled exceptions in the middleware pipeline
+- Captures exceptions handled by the framework `UseExceptionHandler` and Error page display
+- Captures process-wide unhandled exceptions (AppDomain)
+- Captures `LogError` or `LogCritical`
+- Any event sent will include relevant application log messages
+- RequestId as tag
+- URL as tag
+- Environment is automatically set (IHostingEnvironment)
+- Release automatically set (`AssemblyInformationalVersionAttribute`, `AssemblyVersion` or environment variable)
+- Request payload can be captured if opt-in
+- Support for EventProcessors registered with DI
+- Support for ExceptionProcessors registered with DI
+- Supports configuration system (bind to `SentryAspNetCoreOptions`)
+- Server OS info sent
+- Server Runtime info sent
+- Request headers sent
+- HTTP Proxy configuration
+- Request body compressed
+- Event flooding protection (429 retry-after and internal bound queue)
+- Assembly Strong named
+
+## Installation
+
+Add the Sentry dependency:
+
+```powershell {tabTitle:Package Manager}
+Install-Package Sentry.Google.Cloud.Functions -Version {{ packages.version('sentry.dotnet.google-cloud-function') }}
```

Comment:
I noticed the sdk name is `sentry.dotnet.google-cloud-function`
I believe I mixed up at first between `function` and functions` but would be worth double checking if it's aligned everywhere (at least consistent even though the package is `Sentry.Google.Functions` and sdk.name is `sentry.dotnet.google-cloud-function`